sdk/core/System.ServiceModel.Rest/tests/OpenAIClient/OpenAIClientOptions.cs

```diff
@@ -5,6 +5,6 @@
 
 namespace OpenAI;
 
-public class OpenAIClientOptions : RequestOptions
+public class OpenAIClientOptions : InvocationOptions
```

Comment:
Should client options be pipeline construction options or default invocation options?  The interplay of these two still needs to be worked through, I think.